docs/content/en/docs/tasks/packages/metrics-server/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Metrics Server"
+linkTitle: "Add Metrics Server"
+weight: 13
+date: 2022-10-20
+description: >
+  Install/upgrade/uninstall Metrics Server
+---
+
+{{< content "../prereq.md" >}}
+
+
+## Install
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate the package configuration
+   ```bash
+   eksctl anywhere generate package metrics-server --cluster clusterName > metrics-server.yaml
+   ```
+
+1. Add the desired configuration to `metrics-server.yaml`
+
+   Please see [complete configuration options]({{< relref "../../../reference/packagespec/metrics-server" >}}) for all configuration options and their default values.
+
+    Example package file configuring a cluster autoscaler package to run on a management cluster.
+    ```yaml
+    apiVersion: packages.eks.amazonaws.com/v1alpha1
+    kind: Package
+    metadata:
+      name: metrics-server
+      namespace: eksa-packages-<cluster-name>
+    spec:
+      packageName: metrics-server
+      targetNamespace: <namespace-to-install-component>
+      config: |-
+        args:
+          - "--kubelet-insecure-tls"
```

Comment:
For release `0.12` we are not documenting a proper TLS configuration. cc @jiayiwang7 This will likely be a fast follow.